lib/orbital/models/response.rb

```diff
@@ -155,6 +156,13 @@ def self.auth_responses_from_kb_payment_id(kb_payment_id, kb_tenant_id)
         where(:kb_payment_id => kb_payment_id, :kb_tenant_id => kb_tenant_id, :api_call => 'authorize').order(:created_at)
       end
 
+      def self.find_cit_transaction_ref_id(kb_transaction_id, kb_tenant_id)
```

Comment:
Shouldn't this be `find_mit_transaction_ref_id `?

Reply:
The transaction to be found is actually a CIT transaction so I think this name is Ok.